pyproject.toml

```diff
@@ -1,12 +1,6 @@
 [project]
 name = "baybe"
 description = "Bayesian Back End for Design of Experiments"
-authors = [
```

Comment:
I think we should still have "something" here for the author. @Scienfitz, what are your thoughts? 
Options: either we refer to the emdgroup account (that would be probably just "emdgroup") or we write some form of "Merck" (not sure which version would be most appropriate here)

But if you add it, please check if "authors" is still the right field for it or, if for a single author, there is also a separate "author" field or similar. See also my other comment

Reply:
I have now included "Merck KGaA, Darmstadt, Germany" here which is the term that we also use in the License and other.

Reply:
in the meta example they put the ompany name which is now done, thats the best solution for me, imo resolved

Reply:
Yes, sorry for the confusion. What I really meant is "remove **our** names from there", but we need this field filled for proper info PyPI. Thanks! I will also add the same "sorted by last name" thing as discussed in the other PR

Reply:
looking good (also in wheel). As discussed in the other PR, author list will just stay as is